src/config/config.php

```diff
@@ -2,7 +2,7 @@
 
 return [
 
-    /*
+    /**
```

Comment:
Config files are using single `/*` for comments. Look on Laravel config files for example.